understanding/21/label-in-name.html

```diff
@@ -9,22 +9,21 @@
 		<h1>Understanding Label in Name</h1>
 		<section id="intent">
 			<h2>Intent</h2>
-			<p>The intent of this Success Criterion (SC) is to make text that is <em>visually</em> associated with a component also associated with the component <em>programmatically</em>. This helps ensure that people with disabilities can rely on visible labels as a means to interact with the components.</p>
+			<p>The intent of this Success Criterion (SC) is to make text that is <em>visually</em> associated with a component the same as text used to represent the component <em>programmatically</em>. This helps ensure that people with disabilities can rely on visible labels as a means to interact with the components.</p>
```

Comment:
I'm now finding the text biasing towards the programmatic text. We want the visual text to guide the programmatic text, not the other way around. How about
"...with a component also reflected in the text used to represent..."